.github/workflows/lint.yml

```diff
@@ -14,3 +14,20 @@ jobs:
     - uses: astral-sh/ruff-action@v1
       with:
         changed-files: 'true'
+
+  djlint:
+    runs-on: ubuntu-latest
+    name: Lint HTML
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+
+    - name: Install dependencies
+      run: |
+        pip install -U pip
+        pip install djlint
```

Comment:
You will probably encounter the same issue with using pip to install tools globally once this workflow runs on an Ubuntu 24.04 worker in GitHub actions.  Perhaps consider the pipx route also here?

Reply:
I won't since one of the solutions to the global pip install is using the `Set up Python` step before (see: https://github.com/actions/runner-images/issues/10781#issuecomment-2414587300)